base/ctf/msctf/CMakeLists.txt

```diff
@@ -18,9 +14,13 @@ list(APPEND PCH_SKIP_SOURCE
     inputprocessor.cpp
     langbarmgr.cpp
     mlng.cpp
+    msctf.cpp
     range.cpp
     threadmgr.cpp
-    utils.cpp)
+    utils.cpp
+    ${CMAKE_CURRENT_BINARY_DIR}/msctf_stubs.c)
+
+list(APPEND PCH_SKIP_SOURCE)
```

Comment:
Yes, this can be removed, as well as the `PCH_SKIP_SOURCE` entry in the add_library below.

Reply:
Removed.